playwright/_impl/_assertions.py

```diff
@@ -511,15 +511,14 @@ async def to_be_attached(
         timeout: float = None,
     ) -> None:
         __tracebackhide__ = True
+        if attached is None:
+            attached = True
+        attached_string = "attached" if attached else "detached"
         await self._expect_impl(
-            (
-                "to.be.attached"
-                if (attached is None or attached is True)
-                else "to.be.detached"
-            ),
+            ("to.be.attached" if attached else "to.be.detached"),
```

Comment:
not: should we reuse `attached_string` here?

```suggestion
            f"to.be.{attached_string}"
```